cmd/natsdumper/csvwriter/csv_writer.go

```diff
@@ -0,0 +1,197 @@
+package csvwriter
```

Comment:
This is the wrong location for this package. The CMD folder should just be for commands that run things, not supporting packages.